tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/ExceptionReplaySettings.cs

```diff
@@ -15,23 +15,24 @@
 
 namespace Datadog.Trace.Debugger.ExceptionAutoInstrumentation
 {
-    internal class ExceptionDebuggingSettings
+    internal class ExceptionReplaySettings
     {
         public const int DefaultMaxFramesToCapture = 4;
         public const int DefaultRateLimitSeconds = 60 * 60; // 1 hour
         public const int DefaultMaxExceptionAnalysisLimit = 100;
 
-        public ExceptionDebuggingSettings(IConfigurationSource? source, IConfigurationTelemetry telemetry)
+        public ExceptionReplaySettings(IConfigurationSource? source, IConfigurationTelemetry telemetry)
         {
             source ??= NullConfigurationSource.Instance;
             var config = new ConfigurationBuilder(source, telemetry);
 
-            Enabled = config.WithKeys(ConfigurationKeys.Debugger.ExceptionDebuggingEnabled).AsBool(false);
+            Enabled = config.WithKeys(ConfigurationKeys.Debugger.ExceptionDebuggingEnabled).AsBool(false) ||
+                      config.WithKeys(ConfigurationKeys.Debugger.ExceptionReplayEnabled).AsBool(false);
```

Comment:
You should use this pattern: 
```suggestion
            Enabled = config
                         .WithKeys(ConfigurationKeys.Debugger.ExceptionDebuggingEnabled, ConfigurationKeys.Debugger.ExceptionReplayEnabled)
                         .AsBool(false);
```

Reply:
Done in 54b79e765d45303f5eb40737e4461a08eec4ea6d.